prow/overlays/smaug/service-monitor.yaml

```diff
@@ -6,10 +6,12 @@ metadata:
     monitor-component: operate-first
 spec:
   endpoints:
-    - interval: 30s
+    - path: /metrics
       port: metrics
-      scheme: https
+      scheme: http
   namespaceSelector:
     matchNames:
       - opf-ci-prow
-  selector: {}
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: prow
```

Comment:
Do we need this selector? We already have specified that we want to monitor things in the `opf-ci-prow` namespace, what are we hoping to filter out by using this additional label selector?

Reply:
yes, you're right. removing the selector works